### PR TITLE
Cleanup changelog for v8

### PR DESCRIPTION
**Pull request by mpoke**
- Fix typo
- Remove older versions
- Add Previous Versions section

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,479 +58,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
-* (tests) Add [integration ccv tests](https://github.com/cosmos/gaia/blob/main/tests/ics/interchain_security_test.go). Imports Interchain-Security's `TestCCVTestSuite` and implements Gaia as the provider chain.
-
-## [v9.0.0-rc0] - 2023-01-19
-
-* (feat) Add [Interchain-Security](https://github.com/cosmos/interchain-security) [v1.0.0-rc2](https://github.com/cosmos/interchain-security/releases/tag/v1.0.0-rc2) provider module. See the [ICS Spec](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/README.md) for more details.
-* (tests) Add [E2E ccv tests](https://github.com/cosmos/gaia/blob/main/tests/e2e/e2e_gov_test.go#L138). Tests covering new functionality introduced by the provider module to add and remove a consumer chain via governance proposal.
-* (tests) Add [integration ccv tests](https://github.com/cosmos/gaia/blob/main/tests/ics/interchain_security_test.go). Imports Interchain-Security's `TestCCVTestSuite` and implements Gaia as the provider chain.
-
-## [v7.1.0] -2022-10-14
```
Comment (mmulji-ic) [inline CHANGELOG.md:75]: Why scrap v7.x.x - v1.x.x change logs that contribute to v8?

File: CHANGELOG.md [hunk 2]
```diff
@@ -56,479 +58,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
-* (gaiacli) [\#4227](https://github.com/cosmos/cosmos-sdk/issues/4227) Support for Ledger App v1.5
-* (rest) [\#2007](https://github.com/cosmos/cosmos-sdk/issues/2007) Return 200 status code on empty results
-* (rest) [\#4123](https://github.com/cosmos/cosmos-sdk/issues/4123) Fix typo, url error and outdated command description of doc clients.
-* (rest) [\#4129](https://github.com/cosmos/cosmos-sdk/issues/4129) Translate doc clients to chinese.
-* (rest) [\#4141](https://github.com/cosmos/cosmos-sdk/issues/4141) Fix /txs/encode endpoint
+[CHANGELOG of previous versions](https://github.com/cosmos/gaia/blob/main/CHANGELOG.md)
 
 <!-- Release links -->
 
-[Unreleased]: https://github.com/cosmos/gaia/compare/v8.0.0...HEAD
```
Comment (mmulji-ic) [inline CHANGELOG.md:482]: If we keep the release links then they should be grouped with the release text. Having a list at the bottom of the file doesn't feel useful or ergonomic.

File: CHANGELOG.md (hunk 1 above)
Comment (mpoke) [reply CHANGELOG.md:75]: Because they are in main and we added a link to that, see `## Previous Versions` section.

File: CHANGELOG.md (hunk 2 above)
Comment (mpoke) [reply CHANGELOG.md:482]: Yeah, both ibc-go and cosmos-sdk do it like you say. I just wanted to keep the current format. Not a strong objection here.

File: CHANGELOG.md (hunk 1 above)
Comment (mmulji-ic) [reply CHANGELOG.md:75]: Thats sufficient to remove the noise.

**Review by mmulji-ic — APPROVED** [review]

**Review by sainoe — APPROVED** [review]

**Pull request merged**